style.css

```diff
@@ -44,13 +48,38 @@ body {
   justify-content: space-around;
   align-items: center;
   border-bottom: 1px solid hsla(209, 75%, 50%, 8%);
```

Comment:
better to use a variable, as it will be easy to reuse/change.

Reply:
removed the property, not using border anymore.